cmd/bad-key-revoker/main.go

```diff
@@ -388,7 +388,7 @@ func (bkr *badKeyRevoker) invoke() (bool, error) {
 type Config struct {
 	BadKeyRevoker struct {
 		DB        cmd.DBConfig
-		DebugAddr string
+		DebugAddr string `validate:"hostname_port"`
 
 		TLS       cmd.TLSConfig
 		RAService *cmd.GRPCClientConfig
```

Comment:
Aren't these fields required?

Reply:
Summary of our offline conversation: individual structs are not validated as non-nil. You require them tagging their constituent fields in such a way that the struct will failed validation. Similarly, you have to omit `validate:"-"` struct valued fields with tagged sub-fields in contexts where they're optional. In a way, if any of a structs fields are tagged in such a way that they're required, that whole struct now becomes required unless it's omitted in that context.